### DISCUSSION TITLE
Filebeat Netflow Module

### DISCUSSION THREAD

**Discussion by gobrien1231** [Security-Onion-Solutions/securityonion#6112] [Unsupported Versions] [answered]
Hi there. I was wondering if anyone here has successfully integrated SO Kabana with the filebeat netflow module? Specifically looking for any pointers how the configuration would be tailored to run persistently in SO. 

I did network installation of SO on Ubuntu 18.04, SO version 2.3.8.0, standalone architecture

**Comment by dougburks** [comment, marked as answer]
https://github.com/Security-Onion-Solutions/securityonion/discussions/5781
https://docs.securityonion.net/en/latest/filebeat.html?#modules
